index.html

```diff
@@ -839,6 +839,9 @@ <h2>
           not [=Document/fully active=], then return <a>a promise rejected
           with</a> an {{"AbortError"}} {{DOMException}}.
           </li>
+          <li>If |document|'s [=Document/visibility state=] is not `"visible"`,
```

Comment:
I think we need to include this for updating a Rec:

```suggestion
          <li class="addition proposed">If |document|'s [=Document/visibility state=] is not `"visible"`,
```

Reply:
Hrm, I merged this initially, but now am less sure. Is the Payment Request repo not now in a '1.1' state (see https://w3c.github.io/payment-request/) where published changes !== changes to a Rec? 

For example, we didn't mark https://github.com/w3c/payment-request/pull/982/files as 'addition proposed' as far as I know. (Though that may also have been an oversight of course!).

WDYT? (cc @ianbjacobs too)

Reply:
I checked with @ianbjacobs this morning and he agreed, so I reverted this commit. I'm going to land the spec change without it, but if you have concerns @marcoscaceres then lets definitely follow up on this and all the other commits that have landed in 1.1 and see if we should mark them all :)